src/types/sections.ts

```diff
@@ -0,0 +1,109 @@
+import { BaseItemKind, SortOrder } from '@jellyfin/sdk/lib/generated-client';
+import { ItemSortBy } from '@jellyfin/sdk/lib/models/api/item-sort-by';
+import { CardOptions } from './cardOptions';
+import { SectionsView } from './libraryTabContent';
+
+export interface ParametersOptions {
+    sortBy?: ItemSortBy[];
+    sortOrder?: SortOrder[];
+    includeItemTypes?: BaseItemKind[];
+    isAiring?: boolean;
+    hasAired?: boolean;
+    isMovie?: boolean;
+    isSports?: boolean;
+    isKids?: boolean;
+    isNews?: boolean;
+    isSeries?: boolean;
+    isInProgress?: boolean;
+    IsActive?: boolean;
+    IsScheduled?: boolean;
+    limit?: number;
+    imageTypeLimit?: number;
+}
+
+export enum SectionApiMethod {
+    ResumeItems = 'resumeItems',
+    LatestMedia = 'latestMedia',
+    NextUp = 'nextUp',
+    RecommendedPrograms = 'RecommendedPrograms',
+    LiveTvPrograms = 'liveTvPrograms',
+    Recordings = 'Recordings',
+    RecordingFolders = 'RecordingFolders',
+}
+
+export enum SectionType {
+    ContinueWatchingMovies = 'continuewatchingmovies',
+    LatestMovies = 'latestmovies',
+    ContinueWatchingEpisode = 'continuewatchingepisode',
+    LatestEpisode = 'latestepisode',
+    NextUp = 'nextUp',
+    LatestMusic = 'latestmusic',
+    RecentlyPlayedMusic = 'recentlyplayedmusic',
+    FrequentlyPlayedMusic = 'frequentlyplayedmusic',
+    ActivePrograms = 'ActivePrograms',
+    UpcomingEpisodes = 'UpcomingEpisodes',
+    UpcomingMovies = 'UpcomingMovies',
+    UpcomingSports = 'UpcomingSports',
+    UpcomingKids = 'UpcomingKids',
+    UpcomingNews = 'UpcomingNews',
+    LatestRecordings = 'LatestRecordings',
+    RecordingFolders = 'RecordingFolders',
+    ActiveRecordings = 'ActiveRecordings',
+    UpcomingRecordings = 'UpcomingRecordings'
+}
+
+export interface Section {
+    name: string;
+    type: SectionType;
+    apiMethod?: SectionApiMethod;
+    itemTypes: string;
+    parametersOptions?: ParametersOptions;
+    cardOptions: CardOptions;
+}
+
+export const DEFAULT_Movies_Suggestions_Tab_SECTIONS: SectionsView = {
```

Comment:
This naming convention is kind of strange... what about something like `MovieSuggestionsSectionsView`?